docs/development.md

```diff
@@ -239,3 +253,43 @@ these steps:
 ##### Unmanaged and Hybrid Cleanup
 
 Follow the [cleanup instructions in the user guide](userguide.md#cleanup).
+
+## Test Your Changes
+
+In order to test your changes on your own kubernetes cluster, you'll need
+to build an image and a deployment config.
+
+### Build Federation Container Image
+
+Run the following command using the committed Dockerfile to create a container
+image to use for deployment:
+
+```bash
+docker build . -f Dockerfile.controller -t <container-registry>/<username>/federation-v2:test
+docker push <container-registry>/<username>/federation-v2:test
+```
+
+If intending to use the docker hub `docker.io` as the `<container-registry>` to push
+the federation image to, make sure to login to the local docker daemon
+to ensure credentials are available for push:
+
+```bash
+docker login --username <username>
+```
+
+### Create Deployment Config
+
+Run the following command to build the deployment config `hack/install.yaml`
+that includes all the necessary kubernetes resources:
+
+```bash
+kubebuilder create config \
```

Comment:
The generated yaml will cause errors with kube < 1.12 unless the type fields of openapischema structs are removed for resources that define subresources.  The fix is as follows:

https://github.com/kubernetes-sigs/federation-v2/pull/160/files#diff-1ed12d3b5689c7a45b1c21e93e23e3d3R107

Reply:
Added a note about this.